site/app/controllers/student/TeamController.php

```diff
@@ -497,6 +497,28 @@ public function setTeamName($gradeable_id): RedirectResponse {
             $_POST['team_name'] = null;
```

Comment:
you may want to put your logic in an else statement after this check, if the user does not input a team name then no point in doing validation on it